Tools/WinMLRunner/src/EventTraceHelper.cpp

```diff
@@ -0,0 +1,559 @@
+#include "EventTraceHelper.h"
+#include <direct.h>
+
+#define LOGSESSION_NAME _T("winml")

```

Comment:
>_T [](http://example.com/codeflow?start=24&length=2)

_T macro was useful in the old days when unicode wasnt supported, why cant winml just be whar_t?

Reply:
Later on you assume it is wchar_t via sizeof(wchar_t)

Reply:
Sure will fix